app/constants/support.ts

```diff
@@ -0,0 +1,2 @@
+export const WHATSAPP_CONTACT_NUMBER = '+5599985049935'
```

Comment:
should be +503 6983 5117

Reply:
Oh yeah, of course. I just added mine as an example

Reply:
> Oh yeah, of course. I just added mine as an example
you'll risk getting a lot of spam ;)